content/en/blog/pop-joins-falco.md

```diff
@@ -0,0 +1,58 @@
+
+ ### POP's Plan and Focus
+
+ #### Contribution - In order to continue to add contributors, we will:
+
+ - Help explain what it means to contribute to Falco.  If you have a talent for creating Falco rulesets or you want to work on Falco core, I want to help you get there.
+
+ - Make sure folks are welcome when contributing to Falco.  We accomplish this by being available to assist in getting started, answering questions and linking contributors with subject matter experts in order to better encourage contribution
+
+ - Show gratitude for those who are key contributors to the project. Sometimes the easiest way to do this is to say “thank you” or highlight where someone’s contribution is benefiting end users and companies adopting Falco for runtime security.
+
+ - Highlight key contributions to the community (i.e Contributors of the month, Falco welcome kits for those making an impact or who have done their first PR)
+
+ - Make it fun to be part of the Falco project and celebrate how incredible this technology is!  It’s fun to contribute to something that is a great project. I want to build more relationships and foster some amazing technical breakthroughs.
+
+
+ #### Adoption -  Work with current adopters and find more end users using Falco
+
+
+ - Work within the cloud-native community to see where Falco is being used and shepherd their experience so that they also are also advocates for the power of Falco!
+
+ - Assist current adopters with Falco and ensure they have a sounding board for their integration and use of Falco so that their adoption grows even further!
+
+ #### Graduation - Help the project grow from Incubated to Graduated CNCF project
+
+ - Work under the guidance of the CNCF in order to ensure Falco is compliant with the requirements to graduate.
+
+ - Ensure target requirements are being met in order to enrich the user experience in a way that makes the project’s graduation a no-brainer
+
+ My goal as the Lead of Open Source Community and Ecosystem is to spread even more awareness around the Falco project and the magnificent work the team is doing!  We want YOU the community to join the Falco project, tell us how you are using Falco, contribute in places like the Falco core engine, rulesets, integrations, etc.  We would like to make it easy for you to contribute. We want to celebrate your contributions!  We want you to use and triumph in using the Falco engine, rules, and outputs. Use the subprojects like sidekick, event generator and docs page, let us know what would make your experience even better.  If you are a company or technology partner that is using Falco… LET US KNOW, let’s help you adopt the power of Falco to meet your end goals!
+
+ Reach out to me on twitter @danpopnyc or on the [Falco slack](https://kubernetes.slack.com/archives/CMWH3EH32)
```

Comment:
a lot of contributors have put a lot of hours into https://github.com/falcosecurity/community 

maybe we could add more contact detail there?